### DISCUSSION TITLE
Performance impact / bundle size

### DISCUSSION THREAD

**Discussion by AbdallahAlhaddad** [heroui-inc/tailwind-variants#101] [Q&A] [answered]
Hello all, I was wondering about the runtime performance impact and the bundle size increase when using [tailwind-variants](https://www.tailwind-variants.org/). Is there any analysis or benchmarks with regards to this topic?

**Comment by troymcginnis** [comment]
Wondering the same thing here. @AbdallahAlhaddad Have you received any insight on this?

**Comment by mskelton** [comment, marked as answer]
https://github.com/nextui-org/tailwind-variants/discussions/133 is a discussion we keep open with the latest benchmarks.

**Comment by cihad** [comment]
https://bundlejs.com/?q=tailwind-variants&treeshake=%5B%7B+tv+%7D%5D

```ts
export { tv } from "tailwind-variants";
```

```
Bundle size is 26.6 kB -> 9.52 kB (gzip)
```

What's going on?

**Comment by jrgarciadev** [comment]
Improved this a lot on v2 https://github.com/heroui-inc/tailwind-variants/releases/tag/v2.0.0
